LondonBritishLibrary/orient/BLorient13268.xml

```diff
@@ -0,0 +1,164 @@
+                            <msItem xml:id="ms_i1.1.5">
+                                <locus from="15r" to="22v"/>
+                                <title type="incomplete">Psalm 77,57 to Psalm 88,2</title>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2">
+                            <locus from="23ra" to="28va"/>
+                            <title ref="LIT2509Weddas" type="incomplete"/>
+                            <note>Fragments</note>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3">
+                            <locus from="28va" to="30vb"/>
+                            <title ref="LIT1113Anqasa" type="incomplete"/>
+                        </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">30</measure>
+                                    <measure unit="page" type="blank">1</measure><locus target="#1r"/>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>182</height>
+                                        <width>125</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="22"><locus from="1v" to="22v"/></layout>
+                                <layout columns="2" writtenLines="25"><locus from="23r" to="30v"/></layout>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Careful handwriting.</desc>
+                                <date notBefore="1900" notAfter="1978" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc> 
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="1v" to="2v"/>
+                                    <desc type="GuestText">Salām to the Virgin Mary (fragment).</desc>        
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="unspecified">No binding.</decoNote>
```

Comment:
I do not think `decoNote `is suitable to describe the absence of anything, I think it has more to do with condition.
I also wonder if the leaves are completely loose (unbound) or if the thing lacking are just the covers (seems more probable to me, that the description is that of a disintegrated manuscript partially fallen apart)
```
<binding>
                                <condition key="deficient">covers absent or lost</condition>
                            </binding>
```
NB for most elements no `xml:id` needed unless you refer to the element from elsewhere

Reply:
Strelcyn literally says: "unbound". In another place, he speaks about "original flyleafs".
 I therefore assume that it is a pile of loose sheets.

Reply:
OK then
```
<binding>
                                <condition key="other">unbound</condition>
                            </binding>
```